cmd/pebble/cmd_help.go

```diff
@@ -177,6 +177,10 @@ var helpCategories = []helpCategory{{
 	Label:       "Changes",
 	Description: "view changes and tasks",
 	Commands:    []string{"changes", "tasks"},
+}, {
+	Label:       "Command execution",
```

Comment:
This doesn't feel worthy of a category. We'll probably never have any other commands in it other than exec itself. I suggest moving exec to the Basics one for now, perhaps after "stop", as it's an action as well. Eventually we should split Basics into a couple of categories, once we find a nice axis to split it up.

Reply:
Done.